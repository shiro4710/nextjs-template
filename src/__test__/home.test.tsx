import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Home from "@/pages";

describe("Rendering", () => {
  it("Home", () => {
    render(<Home />);
    expect(screen).toMatchSnapshot();
  });
});
