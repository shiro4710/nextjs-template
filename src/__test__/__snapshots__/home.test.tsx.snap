// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Home 1`] = `Screen {}`;
